src/main/java/com/ibm/crail/storage/blkdev/BlkDevStorageServer.java

```diff
@@ -84,10 +82,9 @@ public void printConf(Logger log) {
 	@Override
 	public StorageResource allocateResource() throws Exception {
 		StorageResource resource = null;
-
+		addr = 0;
```

Comment:
Sorry my fault. We need this to be initialized in the constructor otherwise we always add new blocks with address 0.

Reply:
No problem.  I should have realized that as well.